editor/components/toolbar/index.js

```diff
@@ -9,27 +9,34 @@ import classNames from 'classnames';
 import './style.scss';
 import IconButton from 'components/icon-button';
 
-function Toolbar( { controls } ) {
+function Toolbar( { attributes, setAttributes, controls } ) {
```

Comment:
My first reaction to these changes is that the line is becoming blurred between the concept of a general-purpose toolbar and a block-specific toolbar. We should decide if there's value in the former and whether we can or care to preserve this distinction.